libs/cart/src/models/cart-item-input.ts

```diff
@@ -1,4 +1,4 @@
 export interface DaffCartItemInput {
-  productId: string;
+  productId: number;
```

Comment:
Should be off the Type of DaffProduct.

Reply:
Changed this as well as `product_id` in `DaffCartItem`.